### PR TITLE
updates, impliment loop function

### PR DESCRIPTION
**Pull request by khofaai**
some updates also i've add loop logic in autoplay function

### CODE REVIEW

File: carousel.js [hunk 1]
```diff
@@ -4,61 +4,73 @@
...
+		Slider.autoplay = function () {
+			if (settings.autoplay) {
+				Slider.autoplayInterval = setInterval(function() {
+					Slider.getItemByData(Slider.currentImage).fadeOut(function() {
+						Slider.loop(function() {
+							Slider.getItemByData(Slider.currentImage).fadeIn()
+						});
+					});
+				}, settings.timeout);
+			} else if (Slider.autoplayInterval !== null) {
```
Comment (Salahos21) [inline carousel.js]: Does the else if condition suggests that settings.autoplay == false ?

**Comment by Salahos21** [comment]
Does the else if condition suggests that settings.autoplay == false ?

**Comment by khofaai** [comment]
Yes i dooo :), just in case you changed `autoplay` to false without refreshing the page

**Comment by khofaai** [comment]
@Salahos21 in case my answer is not clear just let me know :)

**Pull request closed without merging**